srcpkgs/cross-x86_64-linux-musl/template

```diff
@@ -39,7 +39,9 @@ nostrip_files="libcaf_single.a libgcc.a libgcov.a libgcc_eh.a
  libgnarl_pic.a libgnarl.a libgnat_pic.a libgnat.a libgmem.a"
 depends="${pkgname}-libc-${version}_${revision}"
 
-if [ "$XBPS_TARGET_WORDSIZE" != "64" ]; then
+if [ "${XBPS_TARGET_MACHINE%-musl}" = i686 ]; then
+	:
+elif [ "$XBPS_TARGET_WORDSIZE" != "64" ]; then
 	broken="64-bit crosstoolchain only available on 64-bit host"
 fi
```

Comment:
Is there anything special about `i686` here? We could just remove this limitation.

Reply:
I don't know, to be honest :-p